Sources/SmileID/Classes/Networking/Models/v2/Metadata.swift

```diff
@@ -86,8 +107,33 @@ public enum DocumentImageOriginValue: String, Codable {
     case cameraAutoCapture = "camera_auto_capture"
     case cameraManualCapture = "camera_manual_capture"
 }
-    
+
 public enum CameraFacingValue: String, Codable {
-    case front
-    case back
+    case frontCamera = "front_camera"
+    case backCamera = "back_camera"
+}
+
+public extension UIDevice {
+    var isSimulator: Bool {
+        TARGET_OS_SIMULATOR != 0
+    }
+
+    var modelName: String {
+        var systemInfo = utsname()
+        uname(&systemInfo)
+        let machineMirror = Mirror(reflecting: systemInfo.machine)
+        let identifier = machineMirror.children.reduce("") { identifier, element in
+            guard let value = element.value as? Int8, value != 0 else { return identifier }
+            return identifier + String(UnicodeScalar(UInt8(value)))
+        }
+        return identifier
```

Comment:
I don't know if you have seen the format returned by this yet but I'm thinking we can return a more friendly model name that's easy to understand: e.g. `iPhone 14 Pro` instead of `iPhone15,2`